test/e2e/framework/pod/dial.go

```diff
@@ -0,0 +1,206 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+Copyright 2021 Intel Corporation.
```

Comment:
not much into licensing, but I can't see any other file with a copyright that is no "The kubernete authors"

Reply:
I can change that.